python/test/unit/function/test_interpolation.py

```diff
@@ -218,3 +219,19 @@ def test_mixed_interpolation():
     v = Function(FunctionSpace(mesh, ufl.MixedElement([A, B])))
     with pytest.raises(RuntimeError):
         v.interpolate(lambda x: (x[1], 2 * x[0], 3 * x[1]))
+
```

Comment:
Maybe we should have the following tests:
- vector-DG <-> Nedelec
- vector-DG <-> RT
- CG -> DG
- Same space to same space (copy-assign)